code/Data exploration.R

```diff
@@ -199,20 +254,22 @@ data$SUMMER <- if_else(data$SEASON == 'SUMMER',1,0)
 data$AUTUMN <- if_else(data$SEASON == 'AUTUMN',1,0)
 data$WINTER <- if_else(data$SEASON == 'WINTER',1,0)
```

Comment:
[no change required]: I prefer clarifying in the column name that a column is a flag (e.g., `summer_flag`)

Reply:
Will make the change 👍